packages/ui/src/components/Styleguide/Styleguide.tsx

```diff
@@ -0,0 +1,203 @@
+            <div>
+              <div data-h2-background="base(quinary.lightest)" />
+              <div data-h2-background="base(quinary.lighter)" />
+              <div data-h2-background="base(quinary.light)" />
+              <div data-h2-background="base(quinary)" />
+              <div data-h2-background="base(quinary.dark)" />
+              <div data-h2-background="base(quinary.darker)" />
+              <div data-h2-background="base(quinary.darkest)" />
+            </div>
+            <div>
+              <div data-h2-background="base(foreground.lightest)" />
+              <div data-h2-background="base(foreground.lighter)" />
+              <div data-h2-background="base(foreground.light)" />
+              <div data-h2-background="base(foreground)" />
+              <div data-h2-background="base(foreground.dark)" />
+              <div data-h2-background="base(foreground.darker)" />
+              <div data-h2-background="base(foreground.darkest)" />
+            </div>
+            <div>
+              <div data-h2-background="base(background.lightest)" />
+              <div data-h2-background="base(background.lighter)" />
+              <div data-h2-background="base(background.light)" />
+              <div data-h2-background="base(background)" />
+              <div data-h2-background="base(background.dark)" />
+              <div data-h2-background="base(background.darker)" />
+              <div data-h2-background="base(background.darkest)" />
+            </div>
+          </div>
+        </div>
+        <div
+          data-h2-background="base:children[>div](background)"
+          data-h2-display="base(grid)"
+          data-h2-grid-template-columns="base(1fr 1fr)"
+          data-h2-margin="base(x1, 0)"
+          data-h2-padding="base:children[>div](x1)"
+        >
+          <div data-h2="light">
+            <div
+              data-h2-background="base(background.dark)"
+              data-h2-padding="base(x1)"
+              data-h2-border="base(1px solid background.darkest)"
+              data-h2-color="base(background.darkest)"
+              data-h2-radius="base(5px)"
+            >
+              <p data-h2-font-weight="base(bold)">
+                This is a sample well title.
+              </p>
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Accusamus laudantium, ipsum esse voluptas voluptate sequi facere
+                vitae quo quia. Deleniti, itaque? Reiciendis recusandae repellat
+                animi voluptatem explicabo aliquam neque asperiores.
+              </p>
+            </div>
+          </div>
+          <div data-h2="dark">
+            <div
+              data-h2-background="base(background.dark)"
+              data-h2-padding="base(x1)"
+              data-h2-border="base(1px solid background.darkest)"
+              data-h2-color="base(background.darkest)"
+              data-h2-radius="base(5px)"
+            >
```

Comment:
Looks like we also have a colour contrast issue here. Currently it is `3.63:1`

![Screenshot 2023-09-20 110429](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/c16e5775-f081-45f0-8693-3df2bc4231f4)

Reply:
This fella will be updated as a part of the message wells later, but I'll note it down to be sure.